src/util.h

```diff
@@ -202,6 +202,7 @@ class ArgsManager
     void ParseParameters(int argc, const char*const argv[]);
     void ReadConfigFile(const std::string& confPath);
     std::vector<std::string> GetArgs(const std::string& strArg);
+    std::vector<std::string> GetUniqueArgs(const std::string& strArg);
```

Comment:
+1 to comment.

I can imagine this having wider use, so I don't think adding a ArgsManager member function is a problem.